.github/workflows/ci.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       - name: End to end tests
         run: |
           DELTA_BIN=target/${{ matrix.target }}/release/delta
-          ./tests/test_raw_output_matches_git_on_full_repo_history $DELTA_BIN
+          ./tests/test_raw_output_matches_git_on_full_repo_history $DELTA_BIN &&
           ./tests/test_deprecated_options $DELTA_BIN > /dev/null
```

Comment:
I thought that this fix was correct, but the build doesn't look like I was expecting: https://github.com/dandavison/delta/runs/1493996100

<table><tr><td><img width=300px src="https://user-images.githubusercontent.com/52205/101062586-cfd5e980-355f-11eb-9719-6fb9c60f200e.png" alt="image" /></td></tr></table>

Reply:
I believe the combination of the two fixes above is what we want.